pkg/reconciler/knativeeventing/common/sinkbindingselectionmode.go

```diff
@@ -73,3 +77,21 @@ func SinkBindingSelectionModeTransform(instance *eventingv1beta1.KnativeEventing
 		return nil
 	}
 }
+
+func sinkBindingSelectionModeFromWorkloadOverrides(instance *eventingv1beta1.KnativeEventing) string {
+	for _, workloadOverride := range instance.Spec.Workloads {
+		if workloadOverride.Name == "eventing-webhook" {
+			for _, envRequirement := range workloadOverride.Env {
+				if envRequirement.Container == "eventing-webhook" {
```

Comment:
Do we have to pin the contain name to `eventing-webhook` only?

Reply:
That's a good point. In the eventing deployments for the eventing-webhook, I saw that the container is named that way (https://github.com/knative/eventing/blob/466d123181f39f90a5f38583445910ffe61cbb0d/config/core/deployments/webhook.yaml#L53-L91). But it probably doesn't hurt to check this on all containers.